src/Model.cpp

```diff
@@ -436,11 +436,11 @@ static void my_model(std::vector<double> parameter_set, std::vector<::EERAModel:
 	if(seedlist.seedmethod != "background"){
 	//	size_t k = sizeof(seed_pop);
 	//	unsigned int startdist[k];
-		std::vector<int> startdist = {0,0,0,0,0,0};
-		startdist[(int) gsl_ran_flat(r, 0, 6)] = seedlist.nseed;
+		std::vector<int> startdist = {0,0,0,0,0,0,0};
+		startdist[(int) gsl_ran_flat(r, 0, n_agegroup-1 )] = seedlist.nseed;
 	//	gsl_ran_multinomial(r, k, startdz, seed_pop,startdist); //distribute the diseased across the older age categories
 
-		for ( int age =1; age < (n_agegroup-1); ++age) {
+		for ( int age =0; age < (n_agegroup-1); ++age) {
 			poparray[age][0] -=  startdist[age-1];//take diseased out of S
```

Comment:
This loop has a problem. On the first iteration, when `age = 0`, the expression `startdist[age - 1]` will access invalid memory, resulting in `poparray[0][0]` and `poparray[0][3]` containing garbage data.

Reply:
should be: 		
for ( int age =0; age < (n_agegroup-1); ++age) {
			poparray[age][0] -=  startdist[age];//take diseased out of S
			poparray[age][3] +=  startdist[age];//put diseased in I	
		}	

Interesting. I thought I changed this already.